.github/workflows/ci.yml

```diff
@@ -40,56 +39,59 @@ jobs:
         with:
           python-version: '3.10'
           architecture: 'x64'
-      - name: Install pylint
-        run: pip install -r dev_tools/requirements/pylint.env.txt
+      - name: Install requirements
+        run: pip install -r dev_tools/requirements/envs/pylint.env.txt
       - name: Lint
         run: check/pylint
+  pytest-minimum:
+    name: Pytest minimum
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install requirements
+        run: |
+          pip install -r dev_tools/requirements/envs/pytest.env.txt
+          pip install cirq-core~=0.15.0
+      - name: Pytest check
+        run: check/pytest
+        shell: bash
   pytest:
     name: Pytest
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # On each operating system, check latest version of python and cirq
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        python-version: [ '3.10' ]
-        cirq-version: [ '~=1.0' ]
-        # Also check least-supported versions (linux only)
-        include:
```

Comment:
What's the minimum python version we support now?

Reply:
Instead of abusing the strategy matrix, I moved the "least supported" test to a new job `pytest-minimum`. There, I bumped the minimum python version for shady reasons:

 - The "minimum" environment uses cirq 0.15, which supports python 3.8 so we could run the job on python 3.8
 - however, the way the job works is first you install the full, modern, pinned environment which requires python 3.9 because cirq 1.2 requires python 3.9
 - After the full environment, we install the older version of cirq. But that step would fail with the old python

The "correct" fix would be to pin a minimal environment. The correct way to do this would be to have another Dockerfile that uses python 3.8

Reply:
I have unbumped the minimum python version by doing a correct fix